lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_shutdown_emulator_action.rb

```diff
@@ -0,0 +1,48 @@
+module Fastlane
+  module Actions
+    class AndroidShutdownEmulatorAction < Action
+      def self.run(params)
+        helper = Fastlane::Helper::Android::EmulatorHelper.new
+        helper.shut_down_emulators!(serials: params[:serial])
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Shuts down Android emulators'
+      end
+
+      def self.details
+        description
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :serial,
+                                       env_name: 'FL_ANDROID_SHUTDOWN_EMULATOR_SERIAL',
+                                       description: 'The serial(s) of the emulators to shut down. If not provided (nil), will shut them all down',
+                                       type: Array,
```

Comment:
Yeah I actually wondered about this. Ended up using singular because I figured in 95% of cases the call site will only provide either `nil` or a single `serial`, and providing multiple serials (but a subset of all of them) would be a rather rare case.

But I guess using `serials` make more sense for consistency with the param being an `Array` after all. Will update.

Reply:
Fixed in 7c152d3705fa050b5f50196a60a7c8bc4000441e (then updated call site in https://github.com/wordpress-mobile/WordPress-Android/commit/b6346ab0eaf45891c8c66e47301b6529a93807b8